client/src/components/ReviewTile.js

```diff
@@ -12,11 +14,77 @@ const ReviewTile = (props) => {
     }
   })
 
+  const handleClick = async (event) => {
+    if(props.userId) {
```

Comment:
do we have an `else` for this if? otherwise it seems like the click wont do anything if they arent logged in. 

I would say you can either display an error message if not logged in, OR you can simply note display the vote buttons if they are not logged in

Reply:
In the TVDetails component, we have a message saying the user must be logged in to vote if they are not logged in.  Should that message only appear after a button is clicked, or is it okay to have that message always be displayed when the user is not logged in?